wgpu-hal/src/gles/conv.rs

```diff
@@ -115,7 +115,7 @@ impl super::AdapterShared {
             Tf::EacRg11Unorm => (glow::COMPRESSED_RG11_EAC, glow::RG, 0),
             Tf::EacRg11Snorm => (glow::COMPRESSED_SIGNED_RG11_EAC, glow::RG, 0),
             Tf::Astc { block, channel } => match channel {
-                AstcChannel::Unorm => match block {
+                AstcChannel::Unorm | AstcChannel::Hdr => match block {
```

Comment:
Shouldn't the formats be different for hdr?

Reply:
From [GL spec](https://registry.khronos.org/OpenGL/extensions/KHR/KHR_texture_compression_astc_hdr.txt), astc_ldr decode to vec4\<f16\> or 8 bit sRGB, astc_hdr only decode to vec4\<f16\>, only two types of formats are available for astc: `_RGBA_ASTC_XXX_KHR ` and `_SRGB8_ALPHA8_ASTC_XXX_KHR`.   `_SRGB8_ALPHA8_ASTC_XXX_KHR` cannot apply to astc_hdr
<img width="613" alt="Screen Shot 2022-09-22 at 05 13 30" src="https://user-images.githubusercontent.com/1001342/191612794-6de84b02-fffd-44d2-b23a-6fc0920603c3.png">

Reply:
This is the most bizarre thing to me.

GL is always full of surprises